app/javascript/tests/public_health/PatientsTable.test.js

```diff
@@ -0,0 +1,152 @@
+    expect(wrapper.state('selectAll')).toBeFalsy();
+    expect(wrapper.state('jurisdiction_paths')).toEqual({});
+    expect(_.size(wrapper.state('assigned_users'))).toEqual(0);
+    expect(wrapper.state('query').workflow).toEqual('exposure');
+    expect(wrapper.state('query').tab).toEqual(Object.keys(mockExposureTabs)[0]);
+    expect(wrapper.state('query').jurisdiction).toEqual(mockJurisdiction1.id);
+    expect(wrapper.state('query').scope).toEqual('all');
+    expect(wrapper.state('query').user).toEqual(null);
+    expect(wrapper.state('query').search).toEqual('');
+    expect(wrapper.state('query').page).toEqual(0);
+    expect(wrapper.state('query').entries).toEqual(25);
+    expect(_.size(wrapper.state('entryOptions'))).toEqual(5);
+  });
+
+  it('Properly renders dropdown', () => {
+    const wrapper = getExposureWrapper();
+    dropdownOptions.forEach(function(option, index) {
+      expect(wrapper.find(Dropdown.Item).at(index).text()).toEqual(option);
+    });
+  });
+
+  it('Inputting text into search bar calls update table function', () => {
+    const wrapper = getExposureWrapper();
+    const handleSearchChangeSpy = jest.spyOn(wrapper.instance(), 'updateTable');
+    expect(handleSearchChangeSpy).toHaveBeenCalledTimes(0);
+    expect(wrapper.state('query').search).toEqual('');
+    wrapper.find('#search').simulate('change', { target: { id: 'search', value: 'search' } });
+    expect(wrapper.state('query').search).toEqual('search');
+    expect(handleSearchChangeSpy).toHaveBeenCalledTimes(1);
+  });
+
+  it ('Clicking "Close Records" option displays Close Records modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(CloseRecords).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(0).text().includes(dropdownOptions[0])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(0).simulate('click');
+    expect(wrapper.find(CloseRecords).exists()).toBeTruthy();
+  });
+
+  it ('Clicking "Update Case Status" option displays Update Case Status modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(UpdateCaseStatus).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(1).text().includes(dropdownOptions[1])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(1).simulate('click');
+    expect(wrapper.find(UpdateCaseStatus).exists()).toBeTruthy();
+  });
+
+  it ('Clicking "Update Assigned User" option displays Update Assigned User modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(UpdateAssignedUser).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(2).text().includes(dropdownOptions[2])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(2).simulate('click');
+    expect(wrapper.find(UpdateAssignedUser).exists()).toBeTruthy();
+  });
+
+  it('Calls updateAssignedUsers and updateTable methods when component mounts', () => {
+    const instance = getInstance();
```

Comment:
I would remove the `getInstance` method and just call wrapper.instance() here

Reply:
unless the component already mounts with shallow - not sure exactly how this works

Reply:
@sgober I will make this adjustment in the next commit since I see the test still operates as expected. I will note that I created this test based on a very similar one that was created in `Export.test.js` on [this line](https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/tests/public_health/Export.test.js#L112). Should a similar update also be made to the `Export.test.js` file then?

Reply:
Yeah we definitely can - unless it causes the test to fail.  Not sure why I did that originally

Reply:
@sgober I just tried making that adjustment in the `Export.test.js` file and the test passed successfully. I can include that tweak in this PR with my next commit then.